.vscode/launch.json

```diff
@@ -0,0 +1,29 @@
+{
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "type": "node",
+      "request": "launch",
+      "name": "Debug Current Test File with Vitest",
```

Comment:
Does this need to be more clear/configurable for other usecases?

Reply:
Yeah, the fact that this assumes always testing in `apps/prairielearn` is unfortunate. Is there any way around that?

Reply:
I could make two configurations, and the second one is for debugging packages, and has a dropdown?

Reply:
Looks like that's how you do it: https://stackoverflow.com/a/58930746 / https://marketplace.visualstudio.com/items?itemName=augustocdias.tasks-shell-input. Worth doing?

Reply:
Maybe also https://github.com/vitest-dev/vscode is worth exploring? I haven't personally tried that

Reply:
I think the extension is 100% the way to go here, it worked seamlessly for me (at least once I cranked up `vitest.maximumConfigs` to 100).